src/blocks/testimonials/inspector.js

```diff
@@ -0,0 +1,213 @@
+
+const { getComputedStyle } = window;
+
+const applyFallbackStyles = withFallbackStyles( ( node, ownProps ) => {
+	const { backgroundColor, textColor } = ownProps.attributes;
+	const editableNode = node.querySelector( '[contenteditable="true"]' );
+	//verify if editableNode is available, before using getComputedStyle.
+	const computedStyles = editableNode ? getComputedStyle( editableNode ) : null;
+	return {
+		fallbackBackgroundColor: backgroundColor || ! computedStyles ? undefined : computedStyles.backgroundColor,
+		fallbackTextColor: textColor || ! computedStyles ? undefined : computedStyles.color,
+	};
+} );
+
+const Inspector = ( props ) => {
+	const {
+		activeStyle,
+		attributes,
+		backgroundColor,
+		bubbleBackgroundColor,
+		bubbleTextColor,
+		layoutOptions,
+		onSetColumns,
+		onSetGutter,
+		onToggleImages,
+		onToggleRoles,
+		onUpdateStyleName,
+		setBackgroundColor,
+		setBubbleBackgroundColor,
+		setBubbleTextColor,
+		setTextColor,
+		textColor,
+	} = props;
+
+	const {
+		showImages,
+		showRoles,
+	} = attributes;
+
+	const gutterOptions = [
+		{
+			/* translators: abbreviation for small size */
+			label: __( 'S', 'coblocks' ),
+			tooltip: __( 'Small', 'coblocks' ),
+			value: 'small',
+		},
+		{
+			/* translators: abbreviation for medium size */
+			label: __( 'M', 'coblocks' ),
+			tooltip: __( 'Medium', 'coblocks' ),
+			value: 'medium',
+		},
+		{
+			/* translators: abbreviation for large size */
+			label: __( 'L', 'coblocks' ),
+			tooltip: __( 'Large', 'coblocks' ),
+			value: 'large',
+		},
+		{
+			/* translators: abbreviation for largest size */
+			label: __( 'XL', 'coblocks' ),
+			tooltip: __( 'Huge', 'coblocks' ),
+			value: 'huge',
+		},
+	];
+
+	return (
+		<InspectorControls>
+			<PanelBody
+				initialOpen={ false }
+				title={ __( 'Styles', 'coblocks' ) }>
+				<div className="block-editor-block-styles block-editor-block-styles coblocks-editor-block-styles">
+					{ layoutOptions.map( ( style ) => (
+						<div
+							aria-label={ style.label || style.name }
+							className={ classnames(
+								'block-editor-block-styles__item block-editor-block-styles__item',
+								{
+									'is-active': activeStyle === style.name,
+								}
+							) }
+							key={ `style-${ style.name }` }
+							onClick={ () => onUpdateStyleName( style ) }
+							onKeyDown={ ( event ) => {
+								if ( ENTER === event.keyCode || SPACE === event.keyCode ) {
+									event.preventDefault();
+									onUpdateStyleName( style );
+								}
+							} }
+							role="button"
+							tabIndex="0"
+						>
+							{ <img alt="" src={ `/wp-content/plugins/coblocks/dist/images/testimonial-style-${ style.name }${ ! showImages ? '--noimage' : '' }.png` } /> }
+							<div className="block-editor-block-styles__item-label block-editor-block-styles__item-label">
+								{ style.label || style.name }
+							</div>
+						</div>
+					) ) }
```

Comment:
I find this solution interesting as we are essentially re-creating the wheel here hacking around the exisitng styles mechanism to accomplish this.

Reply:
I mean honestly, I am not huge on this custom component because WordPress core is moving away from showing style previews the same way as they had been coming in WordPress 6 expected this year. I just feel like this control might be diverging us from the ways that Gutenberg is doing things. What do you think @jlemay-godaddy?

Reply:
Not sure what the solution would be here? Do we have a block v2 using different styles?

Reply:
Block styles are defined in the block settings as opposed to the block.json. This is basically the v1 feature that is still compatible with v2. You would use styles the same way that we already are in the codebase. 
https://github.com/godaddy-wordpress/coblocks/blob/f190e7e7922a95e35e583a8e55ffd3ae1b42c272/src/blocks/opentable/index.js#L41-L58

New block styles ( in Gutenberg and in WP 6 ) do not have image previews. Here is a screenshot of what the block styles sidebar will transform into. 
![image](https://user-images.githubusercontent.com/30462574/153258895-3521cba2-34de-4609-8a89-4ae0bab2c1d3.png)

If we can use the core mechanism we should. I understand we build something here that is essentially a replacement for that core feature. As Core moves ahead with the style mechanism changes we will start to diverge.